frontend/src/app/core/util/helper.ts

```diff
@@ -1,7 +1,15 @@
-export function isNull(toCheck: any): boolean {
+export function isFalsy(toCheck: any): boolean {
     return toCheck === null || toCheck === undefined || toCheck === '' || toCheck === 0 || toCheck === 0.0 || (toCheck instanceof Array && toCheck.length === 0);
```

Comment:
They're all fine with me, but the current implementation feels like "re-implementing the ECMA spec". You can choose which ever variant you like best.